.github/workflows/ci.yml

```diff
@@ -6,7 +6,7 @@ defaults:
 concurrency:
   group: CI-${{ github.ref }}
   # Queue on all branches and tags, but only cancel overlapping PR burns.
```

Comment:
burns?

Reply:
Likely climbing jargon creeping in. I'm so used to attempts == burns.

Reply:
TIL